gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -425,21 +425,28 @@ message VehiclePosition {
   }
   optional CongestionLevel congestion_level = 6;
 
-  // The degree of passenger occupancy of the vehicle or carriage. This field is still
-  // experimental, and subject to change. It may be formally adopted in the future.
+  // The nominal status of passenger occupancy for the vehicle or carriage.
+  // Individual producers may not publish all OccupancyStatus values. Therefore, consumers
+  // must not assume that the OccupancyStatus values follow a linear scale.
+  // Consumers should represent OccupancyStatus values as the nominal status indicated 
```

Comment:
Should "nominal status" be "nominal states" here? Not sure if that's intended or a typo. It works either way but "states" aligns better with the next sentence, IMHO.